### PR TITLE
🐛(tray) fix pods anti-affinity and missing secrets

### PR DESCRIPTION
**Pull request by jmaupetit**
## Purpose

Swift credentials loaded as environment variables are missing from the Tray's main secret.

## Proposal

- [x] add `RALPH_SWIFT_OS_*` variables in Raph app secret

During test deployments, I also fixed the anti-affinity selector of Ralph's DC as it's a non blue-green application.

### CODE REVIEW

File: src/tray/templates/services/app/secret.yml.j2 [hunk 1]
```diff
@@ -7,6 +7,17 @@ metadata:
   name: "{{ ralph_secret_name }}"
   namespace: "{{ project_name }}"
 data:
+  # Swift
+  RALPH_SWIFT_OS_AUTH_URL: "{{ RALPH_VAULT.RALPH_SWIFT_OS_AUTH_URL | b64encode }}"
```
Comment (lunika) [inline src/tray/templates/services/app/secret.yml.j2:11]: All variable are mandatory ?

Comment (jmaupetit) [reply src/tray/templates/services/app/secret.yml.j2:11]: To connect to swift yes. Maybe we should introduce more flexibility in backends configuration and allow to make them optional. Is this what you have in mind?

Comment (lunika) [reply src/tray/templates/services/app/secret.yml.j2:11]: If they are mandartory, they should not be commented in the main file. WDYT ?

Comment (jmaupetit) [reply src/tray/templates/services/app/secret.yml.j2:11]: They are mandatory only if you use this backend.

Comment (jmaupetit) [reply src/tray/templates/services/app/secret.yml.j2:11]: Sorry you mean mandatory from a jinja/ansible perspective, not Ralph! You are perfectly right, this is not consistent.

Comment (jmaupetit) [reply src/tray/templates/services/app/secret.yml.j2]: This is now fixed in a way simpler way. Thanks @lunika :muscle:

**Review by lunika — APPROVED** [review]

**Pull request merged**